images/kubevirt/Dockerfile.ci

```diff
@@ -0,0 +1,20 @@
+# This Dockerfile is used by CI to test using OpenShift Installer against an KubeVirt cloud.
+# It builds an image containing the kubevirt-install command.
+
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7 AS builder
+WORKDIR /go/src/github.com/openshift/installer
+COPY . .
+RUN hack/build.sh
+
+FROM registry.svc.ci.openshift.org/origin/4.7:cli AS cli
+FROM registry.svc.ci.openshift.org/ocp/4.7:base
+
+COPY --from=cli /usr/bin/oc /bin/oc
```

Comment:
Out of curiosity, what will you be using `oc` for in the CI image? That seems to be the only difference between this image and the standard installer CI image at the moment. Will there be more changes to come?

Reply:
I removed the oc part
We will change it in the future

Reply:
Hmm. Now we have a new CI image is the same as the standard installer CI image. Can we wait on this until there are actual differences in the images?

Reply:
You're right 
I'm closing that PR for now